src/common/mangataHelper.js

```diff
@@ -210,26 +210,23 @@ class MangataHelper {
      * @param {boolean} param0.isPromoted Only list promoted pools if true; this value has to be specified
      * @returns
      */
-    getPools = async ({ isPromoted }) => {
+    getPools = async ({ isPromoted } = {}) => {
         const pools = await this.mangata.getPools();
-        const that = this;
-
-        const filterd = isPromoted ? _.filter(pools, (item) => item.isPromoted) : pools;
+        const filterdPools = isPromoted ? _.filter(pools, (item) => item.isPromoted) : pools;
+        return filterdPools;
+    };
 
-        const formatted = _.map(filterd, (item) => {
-            const firstToken = _.find(that.assets, { id: item.firstTokenId });
-            const firstTokenAmountFloat = (new BN(item.firstTokenAmount)).div(getDecimalBN(firstToken.decimals));
+    processPoolTokenAmountDecimals = (pool) => {
```

Comment:
This function is for `formatting`, as in the original variable name `formatted`. When you name the function you need to mean it. Do not use `process` for function name, as it’s too general.

Reply:
Fixed.